### PR TITLE
Add pyquickhelper

### PR DESCRIPTION
**Pull request by janjagusch**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by janjagusch** [comment]
@xhochy, could you give this a review, please? 👼

File: recipes/pyquickhelper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - decrypt --help
+    - encrypt_file --help
+    - decrypt_file --help
+    # - pyq-sync --help  # currently broken: https://github.com/sdpython/pyquickhelper/issues/340
+  requires:
+    - pip
+
+about:
+  home: http://www.xavierdupre.fr/app/pyquickhelper/helpsphinx/index.html
+  summary: 'Various functionalities: folder synchronization, simple logging function, helpers to generate documentation with sphinx, sphinx extension, to run a command line, to run a notebook...'
```
Comment (chrisburr) [inline recipes/pyquickhelper/meta.yaml:51]: Can you make this a little shorter? The `summary` should be limited to ~80 characters while longer text can be included in the `description` field.

Comment (JanJaguschQC) [reply recipes/pyquickhelper/meta.yaml]: I have changed the summary, according to what it says in the title of the [pyquickhelper homepage](http://www.xavierdupre.fr/app/pyquickhelper/helpsphinx/).

Comment (janjagusch) [reply recipes/pyquickhelper/meta.yaml]: I have changed the summary according to what it says in the title of the [pyquickhelper homepage](http://www.xavierdupre.fr/app/pyquickhelper/helpsphinx/).

**Review by xhochy — APPROVED** [review]

**Pull request merged**